### ISSUE TITLE
Bitcoind version is checked after trying to load the watchonly wallet

### ISSUE THREAD

**Issue by darosior** [wizardsardine/liana#284]
Reported by @cc1 in #272. See https://github.com/wizardsardine/liana/issues/272#issuecomment-1379547330.

**Comment by darosior** [comment]
So the descriptor import on `bitcoind` is failing. It's typical of running a too old `bitcoind` that doesn't support Miniscript. But we do check the version of `bitcoind` when starting up!
https://github.com/wizardsardine/liana/blob/80fb0ecb01f65c15782f215fdda9bc99ad6c78d2/src/bitcoin/d/mod.rs#L573-L577

**Comment by darosior** [comment]
@cc1 Can you confirm you are running a supported `bitcoind` (at least version 24.0) by running `bitcoind -version`?

**Comment by darosior** [comment]
Alright, we actually perform the sanity checks *after* trying to create or load the watchonly wallet. I'll push a fix for the next version.

https://github.com/wizardsardine/liana/blob/80fb0ecb01f65c15782f215fdda9bc99ad6c78d2/src/lib.rs#L209-L214

**Comment by crypto-axis** [comment]
version is 23:

```
cc1@cc1:~$ bitcoind --version
Bitcoin Core version v23.0.0
Copyright (C) 2009-2022 The Bitcoin Core developers

Please contribute if you find Bitcoin Core useful. Visit
<https://bitcoincore.org/> for further information about the software.
The source code is available from <https://github.com/bitcoin/bitcoin>.

This is experimental software.
Distributed under the MIT software license, see the accompanying file COPYING
or <https://opensource.org/licenses/MIT>
```

**Comment by darosior** [comment]
Yeah. The minimum version supported by Liana is 24.0. You can fix this issue by [upgrading to 24.0.1](https://bitcoincore.org/en/download/).
